bin/dasht-query-line

```diff
@@ -245,9 +243,11 @@ dasht-docsets "$@" | while read -r docset; do
 
     { $1 = $1 } # strip whitespace from key
 
+    $2 == "=" {
+      result[$1] = substr($0, index($0, $2) + length($2) + 1)
```

Comment:
I would change the `index($0, $2)` call to `length($1) + 1` for robustness (no problem if `$1` contains `=`).